### PR TITLE
feat(app): rename base -> block for TC copy

### LINKED ISSUE #6086: Thermocycler Run app <> PD terminology
## Background
In the run app the TC "block" is referred to as the "base"
In the v2 api documentation TC commands reference the "block" (ex. set_block_temperature)
In the Protocol Designer the TC "block" is called "block"

## Acceptance criteria 
Update "base" -> "block" in the run app
Locations:
- [ ] TC status on the Pipettes & Modules page
- [ ] The modal for setting a target TC temp on the Pipettes & Modules page
- [ ] TC status on the Run page
- [ ] The modal for setting a target TC temp on the Run page

### PR DESCRIPTION
**Pull request by IanLondon**
# Overview

Closes #6086

# Changelog

- Rename "base" to "block" re TC

# Review requests

Update "base" -> "block" in the run app

Locations:
- [ ] TC status on the Pipettes & Modules page
- [ ] The modal for setting a target TC temp on the Pipettes & Modules page
- [ ] TC status on the Run page
- [ ] The modal for setting a target TC temp on the Run page

# Risk assessment

Low, just a copy change

### CODE REVIEW

**Review by sfoster1 — COMMENTED** [review]
These should probably migrate to tr() tagged constants at the top

**Comment by shlokamin** [comment]
>  tr() tagged constants

@sfoster1 what are those?

**Comment by sfoster1** [comment]
> > tr() tagged constants
> 
> @sfoster1 what are those?

Sorry @shlokamin that's terminology from previous lives using QT. I mean moving string constants to the top of the file as `const`s as we do [here](https://github.com/Opentrons/opentrons/blob/91983894c89271da1c41782f6061cf28c8234725/app/src/components/CalibrateTipLength/AskForCalibrationBlockModal.js#L22-L38), which is done to aid future i18n efforts by gathering string constants to one place so they can be used with platform i18n apis both for gathering constants for translation and substituting translated phraseology for the built-in constants.

File: app/src/components/ModuleControls/index.js [hunk 1]
```diff
@@ -25,18 +29,18 @@ export function ModuleControls(props: Props): React.Node {
           className={styles.temp_data_item}
           current={mod.data.currentTemp}
           target={mod.data.targetTemp}
-          title={
+          title={`${
```
Comment (mcous) [inline app/src/components/ModuleControls/index.js:32]: Is this wrapping template string doing anything?

**Review by mcous — APPROVED** [review]
🖨️

**Review by shlokamin — APPROVED** [review]

File: app/src/components/ModuleControls/index.js (hunk 1 above)
Comment (IanLondon) [reply app/src/components/ModuleControls/index.js:32]: nope, d'oh

**Pull request merged**